aggregate/billing_functions/seqr.py

```diff
@@ -658,11 +660,15 @@ def get_ratios_from_date(
     """
     assert isinstance(dt, datetime)
 
+    if dt < FIRST_CRAM_DATE:
```

Comment:
This function is used in a couple of places, could we move this logic outside this function?

Reply:
Oh right, wait so where would this logic go then? Is there a specific function in seqr.py where it would be more appropriate?

Reply:
I'd implement it in two places (because they have slightly different interpretations):


1. https://github.com/populationgenomics/gcp-cost-control/blob/d797806cee4495f112f50fe25e67daad6900c200/aggregate/billing_functions/seqr.py#L330-L331: No production level joint calls exist before October, so we're unable to distribute the seqr running costs

2. https://github.com/populationgenomics/gcp-cost-control/blob/d797806cee4495f112f50fe25e67daad6900c200/aggregate/billing_functions/seqr.py#L94-L95: Up until this point, we were testing the deploy of seqr, so no proportional cost should be distributed (because CRAMs might actually exist at this stage).